standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -350,7 +350,7 @@ private ControllerServiceMain startLocalController(int controllerId) {
                 .trustStorePasswordPath(keyPasswordFile)
                 .build();
 
-        StoreClientConfig storeClientConfig = StoreClientConfigImpl.withZKClient(zkClientConfig);
+        StoreClientConfig storeClientConfig = StoreClientConfigImpl.withPravegaTablesClient(zkClientConfig);
```

Comment:
This change is unrelated to this PR.

Reply:
Yes, I mentioned in the description that we are hanging the default store used in standalone to be Pravega tables. So all standalone tests exercise our default options

Reply:
Piggybacking changes makes it difficult to track changes, would you mind changing this in a separate issue?

Reply:
I will handle it in a separate issue. I included it here because I was testing the standalone with liststreams and realised that we were using zookeeper based store as default in standalone

Reply:
Can you create the issue and post the number here, please?

Reply:
https://github.com/pravega/pravega/issues/4092